src/pfb/importers/tsv.py

```diff
@@ -136,6 +163,10 @@ def _convert_tsv(node_name, tsv_record, program, project, link_dests):
                 {"dst_id": tsv_record[item], "dst_name": item.split(".")[0]}
             )
             to_del.append(item)
+    
+    for i in relations:
+        if i["dst_id"] == None:
+            i["dst_id"] = "timing"
```

Comment:
i don't understand this, could you add a comment 🙏

Reply:
oh yeah actually this shouldn't be there this was something that I was testing with the PCDC dictionary